backend/src/routes/projectRouter.ts

```diff
@@ -124,6 +124,23 @@ projects.post(
     }
   }
 );
+// Star Project
+projects.post("/v1/projects/:id/star", async (req: Request, res, next) => {
+  await projectController.addStarrer(req.body.user.id, req.body.project);
```

Comment:
we should return some kind of response, either empty or success type so that the call doesn't hang

e.g.
```
POST /api/v1/projects/:id/star
Response 200:
{
  "ok": true
}
```

```
res.status(200).json({ ok: true })
```
something like this?

Reply:
Fixed